src/profile-logic/call-tree.js

```diff
@@ -57,7 +57,7 @@ export class CallTree {
   _rootTotalTime: number;
   _rootCount: number;
   _displayDataByIndex: Map<IndexIntoCallNodeTable, CallNodeDisplayData>;
-  _children: Map<IndexIntoCallNodeTable, CallNodeChildren>;
+  _children: Array<CallNodeChildren>;
```

Comment:
Maybe add a comment stating that the array will be indexed by IndexIntoCallNodeTable values, and explaining that using an array is faster than a Map.